InFrame/Controllers/GradientButton.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  GradientButton.swift
+//  InFrame
+//
+//  Created by 김유진 on 2021/12/01.
+//
+
+import Foundation
+import UIKit
+
+class GradientButton: UIButton{
+    // MARK: - Properties
+    private let viewBounds = UIScreen.main.bounds
+    
+    // MARK: - LifeCycles
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        configureUI()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        configureUI()
+    }
+    
+    // MARK: - Helpers
+    private func configureUI(){
+        let view = UIView(frame: CGRect(x: 0, y: 0, width: viewBounds.width/1.4, height: viewBounds.height/19.80))
+        let gradient = CAGradientLayer()
+
+        gradient.frame = view.bounds
+        gradient.colors = [UIColor(red: 126/255, green: 152/255, blue: 212/255, alpha: 1).cgColor,UIColor(red: 250/255, green: 186/255, blue: 200/255, alpha: 1).cgColor]
```

Comment:
```suggestion
        gradient.colors = [UIColor.rgb(red: 126, green: 152, blue: 212).cgColor, UIColor.rgb(red: 250, green: 186, blue: 200).cgColor]
```
Extension을 활용해 이렇게 작성하시는 게 좋을 것 같아요!

Reply:
적용하겠습니다!!